web_development_101/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -11,28 +12,30 @@ We're going to make a simple implementation of grade-school classic "rock paper
 4. Write a function that plays a single round of Rock Paper Scissors.  The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner of the round like so: `"You Lose! Paper beats Rock"`
    1. make your function case insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation)
 
-   2. __Important note:__ you want to `return` the results of this function call, _not_ `console.log()` them.  To test this function console.log the results:
+2. **Important note:** you want to `return` the results of this function call, _not_ `console.log()` them. To test this function console.log the results:
+
+   ```javascript
```

Comment:
This needs to use `~~~` as the delimiters for code blocks due to the markdown parser we use on the main site. Can you change this back please?